framework/src/org/checkerframework/common/basetype/messages.properties

```diff
@@ -80,9 +80,9 @@ contracts.conditional.postcondition.expression.parameter.name=The conditional po
 lambda.unimplemented=This version of the Checker Framework does not type-check lambda expressions.
 methodref.inference.unimplemented=This version of the Checker Framework does not type-check method references with implicit type arguments.
 
-field.invar.not.found=the field invariant annotation refers to fields not found in a superclass\nfields not found: %s
-field.invar.not.final=the field invariant annotation refers to fields that are not final\nfields not final: %s
-field.invar.not.subtype=the qualifier for field %s is not a subtype of the declared type\nfound: %s\ndeclared type: %s
-field.invar.not.wellformed=the field invariant annotation does not have equal numbers of fields and qualifiers.
-field.invar.not.found.superclass=the field invariant annotation is missing fields that are listed in the superclass field invariant.\nfields not found: %s
-field.invar.not.subtype.superclass=the qualifier for field %s is not a subtype of the qualifier in the superclass field invariant\nfound: %s\nsuperclass type: %s
+field.invariant.not.found=the field invariant annotation refers to fields not found in a superclass\nfields not found: %s
```

Comment:
Hmm, in https://github.com/typetools/checker-framework/blob/master/framework/src/org/checkerframework/common/basetype/messages.properties we definitely use suffixes, e.g. for all the `type.incompatible` keys.
Could it be that it's neither and it just checks for containment? See https://github.com/typetools/checker-framework/blob/master/framework/src/org/checkerframework/framework/source/SourceChecker.java#L1270
I didn't find a place where it splits the message on the `.` and checks for prefix or suffix, nor is there a startsWith or endsWith.